apps/ui/registry/default/ui/group.tsx

```diff
@@ -1,25 +1,43 @@
 import * as React from "react"
 import { mergeProps } from "@base-ui-components/react/merge-props"
 import { useRender } from "@base-ui-components/react/use-render"
+import { cva, type VariantProps } from "class-variance-authority"
 
 import { cn } from "@/lib/utils"
 import { Separator } from "@/registry/default/ui/separator"
 
+const groupVariants = cva(
+  "group flex w-fit [--clip-end:-1rem] [--clip-start:-1rem] *:pointer-coarse:after:min-w-auto [&>*]:focus-visible:z-10",
```

Comment:
[&>*]:focus-visible:z-10 

Added this to the base class to ensure we see the correct focus ring and its not hidden behind other items in the group